FAIL: test_qasm_job_result

<!-- ⚠️ If you do not respect this template, your issue will be closed -->
<!-- ⚠️ Make sure to browse the opened and closed issues -->

### Information

- **Qiskit IBM Provider version**:
- **Python version**:
- **Operating system**:

### What is the current behavior?
This [cron test](https://github.com/Qiskit-Partners/qiskit-ibm/runs/3786817090?check_suite_focus=true#step:5:1588) run fails with
```
======================================================================
FAIL: test_qasm_job_result (test.ibm.test_serialization.TestSerialization)
Test deserializing a QASM job result.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/qiskit-ibm/qiskit-ibm/test/ibm/test_serialization.py", line 121, in test_qasm_job_result
    self._verify_data(result.to_dict(), ())
  File "/home/runner/work/qiskit-ibm/qiskit-ibm/test/ibm/test_serialization.py", line 167, in _verify_data
    self.assertFalse(suspect_keys)
AssertionError: {'results.metadata.active_input_qubits', 'results.metadata.input_qubit_map'} is not false

----------------------------------------------------------------------
```


### Steps to reproduce the problem



### What is the expected behavior?



### Suggested solutions

--- a/test/ibm/test_serialization.py
+++ b/test/ibm/test_serialization.py
@@ -118,7 +118,10 @@ class TestSerialization(IBMTestCase):
         """Test deserializing a QASM job result."""
         result = self.sim_backend.run(self.bell).result()
 
-        self._verify_data(result.to_dict(), ())
+        # Known keys that look like a serialized complex number.
+        good_keys = ('results.metadata.input_qubit_map', 'results.metadata.active_input_qubits')
+
+        self._verify_data(result.to_dict(), good_keys=good_keys)
 
     @slow_test
     def test_pulse_job_result(self):

